client/gutenberg/editor/edit-post/components/layout/index.js

```diff
@@ -57,11 +61,19 @@ function Layout( {
 		'aria-label': __( 'Editor publish' ),
 		tabIndex: -1,
 	};
+
+	const loadRevision = revision => {
+		const { post_content: content, post_title: title, post_excerpt: excerpt } = revision;
+		const postRevision = { ...post, content, title, excerpt };
+		//TODO: what's the better action here? This only loads title, also tried other actions in core/editor without luck
```

Comment:
This is something @youknowriad or @aduth should be able to help.

Reply:
IIf these changes are already saved in the post (in the backend), `updatePost` should be the action to use, if these are "edits" that still need to be saved in the editor, then `editPost` is the right action. I think both of these actions can take only the updated attributes `updatePost( { title } )`

Reply:
I suppose why is that? Do we need to force a reload of the editor to parse post content to blocks?

Reply:
The editor don't reparse the blocks at all at any moment at the moment. If you want to reparse the blocks, you'd have to do it manually (`wp.blocks.parse`) can call the `resetBlocks` action.

Reply:
@youknowriad So if I understand this correctly, we'll want to dispatch 2 actions?
- `updatePost( { title } )` (update title)
- parse the revision's `post_content` then dispatch `resetBlocks`? (update editor content)

Did we need any other actions to update a post's excerpt?

Reply:
I think you should do:

 - `updatePost` to update all the modified properties (title, excerpt, ... and even content) 
 - `resetBlocks` to update the blocks.

(sending `content` in the first action is not strictly necessary but it's a good thing since these two are persisted in two different reducers and we should avoid loss in synchronization) 

This does raise the question of whether we should automatically re-parse the blocks once we call `updatePost` with `content`. We didn't have this use-case in Gutenberg until now and I won't be able to say if t's a good thing/bad thing, how impactful it can be).

Reply:
Thanks @youknowriad we'll give it a shot.

> We didn't have this use-case in Gutenberg until now and I won't be able to say if t's a good thing/bad thing, how impactful it can be).

The core notice around a more recent autosave would be a good spot. The current behavior takes you to the revisions page, but we could provide an option would be to just load that value into the editor.

![screen shot 2018-11-05 at 11 18 08 am](https://user-images.githubusercontent.com/1270189/48021080-869add80-e0ec-11e8-9acf-63055fb5d169.png)